wire-ios-data-model/Source/Proteus/ProteusProvider.swift

```diff
@@ -86,10 +94,14 @@ public class ProteusProvider: ProteusProviding {
 
             return try await proteusServiceBlock(proteusService)
 
-        } else {
+        } else if !proteusViaCoreCrypto {
 
             // remove comment once implementation of proteus via core crypto is done
             return try await keyStoreBlock(keyStore)
+        } else {
+
+            WireLogger.coreCrypto.error("can't access any proteus cryptography service")
+            fatal("can't access any proteus cryptography service")
```

Comment:
Same here. This might cause a crash in production

Reply:
good point but same i would consider this missusage of ProteusProvider